packages/react-components/tests/systems-summary/main-menu-banner.test.tsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+import { MainMenuBanner } from '../../lib';
+import { render } from '@testing-library/react';
+
+test('smoke test', () => {
+  render(<MainMenuBanner bannerUrl={''} isError={false} />);
```

Comment:
Maybe make `bannerUrl` optional? Also suggest renaming it to `imageSrc`, `imageUrl`, `image` or similar.